src/olympia/abuse/actions.py

```diff
@@ -753,6 +758,19 @@ def process_action(self, release_hold=False):
                     )
                 target.update_status()
                 activity_log_action = amo.LOG.UNREJECT_VERSION
+            elif self.previous_decisions.filter(
+                action=DECISION_ACTIONS.AMO_REJECT_VERSION_WARNING_ADDON
```

Comment:
A developer can't appeal a delayed rejection - if it's not cancelled by a reviewer it's followed by a rejection, that they can appeal.  
(And an override can't come from Cinder of delayed rejection, because it's only emitted by reviewer tools)

... right?  Am I missing some case?

Reply:
You're right - I just wanted that to work in case 2nd level approval was ever allowed to "approve content instead" again for add-ons (when I started working on this, I was going to let 2nd level approvers continue to use that in some cases).